app/dashboard/templates/labs.html

```diff
@@ -0,0 +1,176 @@
+                    <span class="score col-sm text-center">{{ tool.vote_score }}</span>
+                    <button class="vote-down col-sm {% if tool.id|is_in_list:profile_down_votes_tool_ids %} active {% endif %}" data-tool-id="{{ tool.id }}"><i class="fas fa-thumbs-down"></i></button>
+                  </div>
+                  <div class="content">
+                    <div class="title">{{ tool.name }}</div>
+                    <div class="description">
+                      {{ tool.i18n_description | safe }}
+                    </div>
+                    {% if tool.link_copy %}
+                      <div class="button_container">
+                        <a class="btn btn-xs btn-success pulseClick" role="button" href="{{ tool.link_url }}">{{ tool.i18n_link_copy }} ></a>
+                      </div>
+                    {% endif %}
+                  </div>
+                </div>
+              {% endfor %}
+            </div>
+          {% endfor %}
+
+          <hr>
+
+          <div class="profile">
+            <h3 class="sub-title dark-text">Meet the Team</h3>
+            <div class="profile-column">
+              <div class="col-md-5">
+                <img class="profile-img" src="{% static 'v2/images/helmet_front.png' %}">
+              </div>
+              <div class="col-md-5 right">
+                <h3 class="sub-title dark-text">Person</h3>
+                <p class="description">A very good person, which helps me in checking the profile cards that I created just now.</p>
+                <a href=" " class="profile-icon"><i class="fab fa-twitter fa-lg"></i></a>
+                <a href=" " class="profile-icon"><i class="fab fa-github fa-lg"></i></a>
+                <a href=" " class="profile-icon"><i class="fab fa-medium fa-lg"></i></a>
+              </div>
+            </div>
+
+            <div class="profile-column">
+              <div class="col-md-5">
+                <img class="profile-img" src="{% static 'v2/images/helmet_front.png' %}">
+              </div>
+              <div class="col-md-5 right">
+                <h3 class="sub-title dark-text">Person</h3>
+                <p class="description">A very good person, which helps me in checking the profile cards that I created just now.</p>
+                <a href=" " class="profile-icon"><i class="fab fa-twitter fa-lg"></i></a>
+                <a href=" " class="profile-icon"><i class="fab fa-github fa-lg"></i></a>
+                <a href=" " class="profile-icon"><i class="fab fa-medium fa-lg"></i></a>
+              </div>
+            </div>                        
+          </div>
+
+          <br><hr>
+          <div>
+            <div class="commmunity-column">
+              <h3 id="{{ actor.slug }}" class="sub-title dark-text">Community</h3>
+              <br>
+              <p>Gitcoin Labs is a community of Ethereum builders, providing research reports and toolkits for busy developers.</p>
```

Comment:
All of the copy throughout these html template files need wrapped in either `{% trans '' %}` or `{% blocktrans %}`